### PR TITLE
Add Pixie as an eBPF project

### PR DESCRIPTION
**Pull request by zasgar**
Signed-off-by: Zain Asgar <zasgar@pixielabs.ai>

Add Pixie as a project. I added Pixie here as a major project since it's deployed in production on a large number of systems. Here is some information about that: 

https://newrelic.com/platform/kubernetes-pixie
https://aws.amazon.com/blogs/opensource/gathering-insights-on-kubernetes-applications-services-and-network-traffic-with-pixie/

If it makes more sense to be an emerging project, please let me know. Thanks!

### CODE REVIEW

File: src/pages/projects.js [hunk 1]
```diff
@@ -246,6 +246,43 @@ const ProjectDescriptions = () => (
...
+            </a>{" "}
+          </p>
+          <p>
+            Pixie is an open source observability tool for Kubernetes
+            applications. Pixie uses eBPF to automatically capture
+            telemetry data without the need for manual instrumentation.
+
+            Developers can use Pixie to view the high-level state of
+            their cluster (service maps, cluster resources, application traffic)
+            and also drill-down into more detailed views (pod state, flame graphs,
```
Comment (dthaler) [inline src/pages/projects.js:279]:
```suggestion
            and also drill down into more detailed views (pod state, flame graphs,
```

**Comment by dthaler** [comment]
> I added Pixie here as a major project since it's deployed in production on a large number of systems. ...
> If it makes more sense to be an emerging project, please let me know. Thanks!

https://ebpf.io/projects#requirements-for-a-project-to-be-listed currently has as one of the requirements for listing as major:

> The project must have more than 50 contributors.

Github currently shows 25 contributors.   Do you have any additional evidence that can be provided about contribitors beyond the set that github shows?   If not, then unless/until the criteria is changed we can list it as Emerging, as I agree it meets all the other criteria.

**Comment by zasgar** [comment]
Thanks @dthaler, I updated it to be "emerging" for now. I'll reach back when we have 50 contributors to move it.

**Review by borkmann — APPROVED** [review]
LGTM, thanks!

**Review by dthaler — APPROVED** [review]

**Pull request merged**